test_fms/fms2_io/test_io_simple.F90

```diff
@@ -0,0 +1,90 @@
+!***********************************************************************
+!*                   GNU Lesser General Public License
+!*
+!* This file is part of the GFDL Flexible Modeling System (FMS).
+!*
+!* FMS is free software: you can redistribute it and/or modify it under
+!* the terms of the GNU Lesser General Public License as published by
+!* the Free Software Foundation, either version 3 of the License, or (at
+!* your option) any later version.
+!*
+!* FMS is distributed in the hope that it will be useful, but WITHOUT
+!* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+!* FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+!* for more details.
+!*
+!* You should have received a copy of the GNU Lesser General Public
+!* License along with FMS.  If not, see <http://www.gnu.org/licenses/>.
+!***********************************************************************
+
+! \brief This is a very simple test of FMS IO.
+
+! \author Ed Hartnett, 6/10/20
```

Comment:
You still need the > after the !.  https://www.doxygen.nl/manual/docblocks.html#fortranblocks
```fortran
!> \brief This is a very simple test of FMS IO.  It opens and closes a netcdf file with the fms2_io.
!> \author Ed Hartnett, 6/10/20
```

Reply:
Sorry to miss that. Fixed now.